changelog/fragments/kubebuilder-dd3942c.yaml

```diff
@@ -0,0 +1,37 @@
+entries:
+  - description: >
+      (go/v3) Pin controller-runtime to v0.7.2
+    kind: change
+    migration:
+      header: (go/v3) Upgrade controller-runtime to v0.7.2
+      body: >
+        In your go.mod file, upgrade `sigs.k8s.io/controller-runtime` to v0.7.2.
+  - description: >
+      (go/v3) Create and bind to a non-default service account ([kubebuilder#2070](https://github.com/kubernetes-sigs/kubebuilder/pull/2070))
+    kind: addition
+    migration:
+      header: (go/v3) Add a `system:controller-manager` ServiceAccount to your project.
+      body: >
+        A non-default ServiceAccount `controller-manager` is scaffolded on `operator-sdk init`,
+        to improve security for operators installed in shared namespaces. To add this ServiceAccount
+        to your project, do the following:
+
+        ```sh
+        # Create the ServiceAccount.
+        cat <<EOF > config/rbac/service_account.yaml
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: controller-manager
+          namespace: system
+        EOF
+
+        # Add it to the list of RBAC resources.
+        echo "- service_account.yaml" >> config/rbac/kustomization.yaml
+
+        # Update all RoleBinding and ClusterRoleBinding subjects that reference the operator's ServiceAccount.
+        find config/rbac -name *_binding.yaml -exec sed -i -E 's/  name: default/  name: controller-manager/g' {} \;
+        
+        # Add the ServiceAccount name to the manager Deployment's spec.template.spec.serviceAccountName.
+        sed -i -E 's/([ ]+)(terminationGracePeriodSeconds:)/\1serviceAccountName: controller-manager\n\1\2/g' config/manager/manager.yaml
```

Comment:
Could we add the example or link with the expected results just in case the commands does not work? 
For example, I can use sed as Linux because I have a gnu pkg installed in the mac os x. However, it is not the common case.

Reply:
We will do Ansible/Helm in another PR right? 
Would not be easier to address then here and have only 1 fragment for all instead of we have one for each language? Shows that it would make the migration guide less verbose. It also can be done in the follow up btw.

Reply:
I think there's some other stuff to be done for ansible/helm regarding configuring the service account. I'd rather do them in a follow-up so the changes can be reverted if necessary.

Reply:
It is fine. The config templates for Ansible/Helm were created by copy and paste from Go. 
So, they should be ==. However, it might have other changes that were not addressed as well.

Reply:
See; https://github.com/operator-framework/operator-sdk/pull/3815/files - it was a PR done to long ago to check if would be possible centralize.